.github/workflows/cpp-packaging.yml

```diff
@@ -92,10 +66,13 @@ jobs:
           github.event.inputs.downloadPublicVersion == '' && github.event.inputs.downloadPreviousRun == ''
         run: echo "::warning ::Verbose build enabled."
 
+      - name: log if expanded matrix enabled
+        if: github.event_name == 'schedule' || github.event.inputs.use_expanded_matrix == '1'
```

Comment:
And one-off workflow checks, like this schedule check, become simply:

`if ${{ needs.prepare_matrix.outputs.use_expanded_matrix }}`

which I feel is easier to maintain.

Reply:
I agree, but I'd like to do this cleanup in a followup.